src/Pos/Txp/Toil/Failure.hs

```diff
@@ -19,7 +20,8 @@ data ToilVerFailure
     = ToilKnown -- ^ Transaction is already in the storage (cache)
     | ToilTipsMismatch { ttmOldTip :: !HeaderHash
                        , ttmNewTip :: !HeaderHash}
-    | ToilOverwhelmed -- ^ Local transaction storage is full -- can't accept more txs
+    | ToilOverwhelmed !Byte -- ^ Local transaction storage is full --
```

Comment:
It was called `ToilOverwhelmed` and I didn't want to change it. On the other hand, for some reason I liked `exhausted` more, hence I called it this way in DLG. I think it's minor anyway.